.github/workflows/python-tox.yml

```diff
@@ -49,6 +51,6 @@ jobs:
         TEST_DB_NAME: wagtail_factories
         TEST_DB_USER: wagtail_factories
         TEST_DB_PASSWORD: secret
-        TOX_ENV: "py${{ matrix.python-version.tox }}-django${{ matrix.django }}-wagtail${{ matrix.wagtail }}-factoryboy${{ matrix.factoryboy }}"
+        TOX_ENV: "python${{ matrix.python-version.tox }}-django${{ matrix.django }}-wagtail${{ matrix.wagtail }}-factoryboy${{ matrix.factoryboy }}"
```

Comment:
This change looks wrong to me. Tox sets its `basepython` based on `pyNM`, not `pythonNM` ([ref](https://tox.wiki/en/latest/config.html#tox-environments)). What's the reasoning here?

Reply:
I think Brian has done that because he changed the tox.ini format

```
envlist =
    python{37,38,39,310}-django32-wagtail{215,216}-factoryboy32
    python{38,39,310}-django{32,40}-wagtail{216,30}-factoryboy32
```

Reply:
I think this must have been working around the fact that older versions of `tox` (<3.14) didn't work with `py310` as a factor ([ref](https://tox.wiki/en/latest/changelog.html#v3-14-0-2019-09-03)), so I've set a `minversion` of `3.14.6` which is the current latest patch version of the `3.14` series and removed the workaround.